src/Approximations/iterative_refinement.jl

```diff
@@ -169,46 +170,52 @@ local `Approximation2D`.
 An ε-close approximation of the given 2D convex set.
 """
 function approximate(S::LazySet{N},
-                     ε::N)::PolygonalOverapproximation{N} where {N<:Real}
+                     ε::N)::PolygonalOverapproximation{N} where N<:Real
 
     # initialize box directions
     pe = σ(DIR_EAST(N), S)
     pn = σ(DIR_NORTH(N), S)
     pw = σ(DIR_WEST(N), S)
     ps = σ(DIR_SOUTH(N), S)
 
-    Ω = PolygonalOverapproximation{N}(S)
+    Ω = PolygonalOverapproximation(S)
 
-    addapproximation!(Ω, pe, DIR_EAST(N), pn, DIR_NORTH(N))
-    addapproximation!(Ω, pn, DIR_NORTH(N), pw, DIR_WEST(N))
-    addapproximation!(Ω, pw, DIR_WEST(N), ps, DIR_SOUTH(N))
+    # add constraints in reverse (i.e., clockwise) order
```

Comment:
Hmm could you plz elaborate on the change? 

Doesn't `# add constraints in reverse (i.e., clockwise) order` conflict with `HPolygon{N}(Ω.constraints)` in `tohrep` constructor, that expects counterclockwise ordering?

Reply:
The `approx_list` became a stack, so you have to add in reverse order. I will rename to `approx_stack` to make that clear.